src/components/WorkflowComponentAdd/WorkflowComponentAdd.tsx

```diff
@@ -1,12 +1,15 @@
-import React, { FC, useCallback } from "react";
```

Comment:
So consider this is the entry point @Darren-Ivey has used for his branch and the drag and drop. I would ideally review this along with the modification that have been made on his branch that will improve on this.